tktermwidget/tkterm.py

```diff
@@ -141,22 +156,20 @@ def down(self, _: Event) -> str:
         """Go down in the history"""
         if self.hi < len(self.historys) - 1:
             self.text.delete(f"{self.index}.0", "end-1c")
-            # Insert the directory
             self.directory()
             # Insert the command
             self.text.insert("insert", self.historys[self.hi].strip())
             self.hi += 1
         else:
             # Clear the command
             self.text.delete(f"{self.index}.0", "end-1c")
-            # Insert the directory
             self.directory()
         return "break"
 
     def left(self, _: Event) -> str:
         """Go left in the command if the command is greater than the path"""
         insert_index = self.text.index("insert")
-        dir_index = f"{insert_index.split('.')[0]}.{len(DIR.format(command=getcwd()))}"
```

Comment:
Indexes only ever have one period

Reply:
Can you give a suggestion change?

Reply:
Just don't make the change :).

Reply:
what?

Reply:
You added `maxsplit=1` even though it doesn't do anything.

Reply:
I used pylint and it added ```maxsplit=1```

Reply:
removed it (I even didn't noticed it add ```maxsplit=1```)